pkg/components/query_tracker.go

```diff
@@ -304,6 +302,20 @@ func (qt *QueryTracker) init(ctx context.Context, ytClient yt.Client) (err error
 		&yt.CreateObjectOptions{
 			Attributes: map[string]interface{}{
 				"name": "queries",
+				"acl": []interface{}{
+					map[string]interface{}{
+						"action":           "allow",
+						"subjects":         []string{"owner"},
```

Comment:
Operator does not create groups and users with "owner" name. Maybe it will be better to give the rights to the "admin" user

https://github.com/ytsaurus/yt-k8s-operator/blob/main/pkg/consts/defaults.go#L3

Reply:
Read that this is a special fictitious user

Reply:
https://ytsaurus.tech/docs/ru/user-guide/storage/access-control#owner

`owner` is a built-in user: https://github.com/ytsaurus/ytsaurus/blob/4979a8c88ac626cdd3e973618c5f1a4d65cca577/yt/yt/server/master/security_server/security_manager.cpp#L3825-L3826

`admins` get access from inherited `access_control_object_namespace` ACL
`owner` will be substituted by creator of a new ACO and will grant access to creator